openwisp_users/migrations/0001_initial.py

```diff
@@ -144,4 +144,8 @@ class Migration(migrations.Migration):
             name='organizationuser',
             unique_together=set([('user', 'organization')]),
         ),
+        migrations.AlterIndexTogether(
```

Comment:
a new migration should be created for this, otherwise existing instances will not get the new index